### PR TITLE
created pl.json, edited en.json and bot.py

### PR DESCRIPTION
**Pull request by MrWurste**
1. Translated commads strings to polish (pl.json).
2. Edited bot.py to give possibility to translate bot's song play messages (async def on_player_play method):
+ skip message when queuer is not in voice,
+ mention to queuer when mentrions are anabled,
+ default 'now plaing in' message,
+ message when autoplalist or autoqueue song is played.
3. Edited en.json to add string lines mentioned above.
4. Translated en.json's new lines in pl.json.

After creating your pull request, tick these boxes if they are applicable to you.

- [x] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.5.3 or higher

----

### Description
Translation of all command strings from en.json file to polish (pl.json) and new messages lines exported from bot.py to en.json with polish translation.
Absence of new lines in another language files doesn't couse eny errors. Default english messages are used.

### CODE REVIEW

**Review by srhinos — APPROVED** [review]
Big fan of these changes! Thanks for the contribution

File: config/i18n/pl.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    "cmd-queue-entry-noauthor": "{0} -- `{1}`",
+    "cmd-clean-invalid": "Nieprawidłowy parametr. Proszę podać ilość wiadomości do wyszukania.",
+    "cmd-clean-reply": "Wyczyszczono {0} wiadomości{1}.",
+    "playlists-noperms": "Nie masz uprawnień do podania listy odtwarzania",
+    "playlists-big": "Lista odtwarzania ma za dużo pozycji ({0} > {1})",
+    "playlists-limit": "Pozycje w liście odtwarzania + zakolejkowane przez Ciebie uwory osiągnęły limit ({0} + {1} > {2})",
+    "karaoke-enabled": "Tryb karaoke jest włączony, proszę spróbować ponownie, gdy będzie wyłączony!",
+    "left-no-owner-guilds": "Opuszczono `{}`, ponieważ właściciel bota nie został tam odnaleziony.",
+	"on_player_play-onChannel_authorNotInChannel_skipWhenAbsent":"Pomijanie następnego utworu {channel}: {title} dodanego przez {author}, gdyż dodającego/dodającej nie ma na kanale!",
+	"on_player_play-onChannel_plaingMention":"{author} - Twój utwór {title} jest teraz odtwarzany w {channel}!",
```
Comment (AutumnClove) [inline config/i18n/pl.json]: plaingMention -> playingMention
Keep it in line with the rest

**Review by AutumnClove — CHANGES_REQUESTED** [review]

**Comment by MrWurste** [comment]
Ok, I'll change it, sorry for mistake.

pt., 22 sty 2021, 09:01 użytkownik Winter <notifications@github.com>
napisał:

File: musicbot/bot.py [hunk 2]
```diff
@@ -488,19 +488,29 @@ async def on_player_play(self, player, entry):
...
-                    player.voice_client.channel.name, entry.title, entry.meta['author'].name)
+                newmsg = self.str.get('on_player_play-onChannel_authorNotInChannel_skipWhenAbsent', "Skipping next song in {channel}: {title} added by {author} as queuer not in voice!").format (
+                    channel=player.voice_client.channel.name,
+                    title=entry.title,
+                    author=entry.meta['author'].name)
                 player.skip()
             elif self.config.now_playing_mentions:
-                newmsg = '%s - your song `%s` is now playing in `%s`!' % (
-                    entry.meta['author'].mention, entry.title, player.voice_client.channel.name)
+                newmsg = self.str.get('on_player_play-onChannel_plaingMention', "{author} - your song {title} is now playing in {channel}!").format (
```
Comment (jayktaylor) [inline musicbot/bot.py]: Fix `plaingMention` to `playingMention`

File: musicbot/bot.py [hunk 3]
```diff
@@ -488,19 +488,29 @@ async def on_player_play(self, player, entry):
...
+                newmsg = self.str.get('on_player_play-onChannel', "Now playing in {channel}: {title} added by {author}!").format(
+                    channel=player.voice_client.channel.name,
+                    title=entry.title,
+                    author=entry.meta['author'].name
+                )
         else:
             # no author (and channel), it's an autoplaylist (or autostream from my other PR) entry.
-            newmsg = 'Now playing automatically added entry `%s` in `%s`' % (
-                entry.title, player.voice_client.channel.name)
+            newmsg = self.str.get ('on_player_play-onChannel_noAuthor_autoplalist', "Now playing automatically added entry {title} in {channel}!").format (
```
Comment (jayktaylor) [inline musicbot/bot.py]: `autoplalist` -> `autoPlaylist`

**Review by jayktaylor — DISMISSED** [review]

**Review by AutumnClove — APPROVED** [review]

**Comment by AutumnClove** [comment]
@MrWurste could you take a glance over the merge I did with review and make sure everything looks okay? I did that at like 2am after an 11 hour work shift lol. Trying to update some PR's and hopefully merge soon and want to see these improvements in the bot.

**Review by AutumnClove — APPROVED** [review]
Love it

**Review by AutumnClove — APPROVED** [review]

**Pull request merged**